gen/gen.go

```diff
@@ -249,10 +249,9 @@ func (g *Gen) writeGoDoc(packageName string, output io.Writer, swagger *spec.Swa
 	return err
 }
 
-var packageTemplate = `// GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
+var packageTemplate = `// Package docs GENERATED BY THE COMMAND ABOVE; DO NOT EDIT
 // This file was generated by swaggo/swag{{ if .GeneratedTime }} at
```

Comment:
since the package name is configurable via ` {{.PackageName}}`, please update it accordingly.

Reply:
> since the package name is configurable via ` {{.PackageName}}`, please update it accordingly.

OK, thanks